connectivity/service/src/test/java/org/eclipse/ditto/connectivity/service/mapping/CloudEventsMapperTest.java

```diff
@@ -0,0 +1,172 @@
+package org.eclipse.ditto.connectivity.service.mapping;
+/*
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import org.apache.kafka.common.protocol.types.Field;
+import org.eclipse.ditto.base.model.common.DittoConstants;
+import org.eclipse.ditto.base.model.exceptions.DittoJsonException;
+import org.eclipse.ditto.base.model.headers.DittoHeaders;
+import org.eclipse.ditto.connectivity.api.ExternalMessage;
+import org.eclipse.ditto.connectivity.api.ExternalMessageFactory;
+import org.eclipse.ditto.json.JsonFactory;
+import org.eclipse.ditto.json.JsonPointer;
+import org.eclipse.ditto.protocol.Adaptable;
+import org.eclipse.ditto.protocol.JsonifiableAdaptable;
+import org.eclipse.ditto.protocol.ProtocolFactory;
+import org.eclipse.ditto.protocol.adapter.DittoProtocolAdapter;
+import org.eclipse.ditto.protocol.adapter.ProtocolAdapter;
+import org.eclipse.ditto.things.model.ThingId;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.singletonList;
+import static org.eclipse.ditto.connectivity.service.mapping.AbstractMessageMapper.extractPayloadAsString;
+import static org.junit.Assert.assertEquals;
+
+public class CloudEventsMapperTest {
+
+  private static final ThingId THING_ID = ThingId.of("thing:id");
+  private static final ProtocolAdapter ADAPTER = DittoProtocolAdapter.newInstance();
+
+  String payload =
```

Comment:
Please use the Java multiline strings for defining the test JSON payloads, e.g.:
```java
String testPayload = """
          {
            "foo":"bar"
          }
          """;
```